addons/account/static/src/js/tours/account.js

```diff
@@ -59,22 +59,22 @@ tour.register('account_tour', {
         content: Markup(_t("Once everything is set, you are good to continue. You will be able to edit this later in the <b>Customers</b> menu.")),
         auto: true,
     }, {
-        trigger: "div[name=invoice_line_ids] .o_field_x2many_list_row_add a:not([data-context])",
+        trigger: "div[name=invoice_line_ids] .o_field_x2many_list_row_add a",
```

Comment:
All the tours "fix" should go in the commit that broke them

Reply:
It's fixing the breaks OWL created and we just discover it now because we activate OWL